### PR TITLE
fixed API GET operations should return results in alphabetical order and respects query parameter ordering

### PR DESCRIPTION
**Pull request by harshkothari410**
- Now if defaultOrder is defined in props then it will take that defaultOrder and return the result as per that. If defaultOrder is not defined then it will take sort order based on model. 

- Fixed all swagger parameter sort option

### CODE REVIEW

File: api/v1/helpers/verbs/findUtils.js [hunk 1]
```diff
@@ -145,9 +145,12 @@ function toSequelizeOrder(sortOrder, modelName) {
 function options(params, props) {
   const opts = u.buildFieldList(params);
 
-  // Specify the sort order
-  const ord = (params.sort ? params.sort.value : null) || props.defaultOrder;
-  opts.order = toSequelizeOrder(ord, props.modelName);
+  // Specify the sort order. If defaultOrder is defined in props then
+  // update sort order otherwise take value from model defination
+  if (props.defaultOrder) {
+    const ord = (params.sort ? params.sort.value : null) || props.defaultOrder;
```
Comment (iamigo) [inline api/v1/helpers/verbs/findUtils.js]: No... it should still sort by params.sort.value even if props.defaultOrder is empty or undefined.

Comment (harshkothari410) [inline api/v1/helpers/verbs/findUtils.js]: Thats good point. I missed it.

**Comment by pallavisingh9** [comment]
Can we have some corresponding tests?

**Comment by pallavi2209** [comment]
Can we have some corresponding tests?

**Pull request merged**